src/Eccube/Resource/template/admin/default_frame.twig

```diff
@@ -32,6 +32,8 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 <link rel="stylesheet" href="{{ asset('assets/css/bootstrap.min.css', 'admin') }}">
 <link rel="stylesheet" href="{{ asset('assets/css/dashboard.css', 'admin') }}">
 {% block stylesheet %}{% endblock %}
+{# TODO:: Block "sub_title" on template "@admin/default_frame.twig" does not exist⇒エラーが発生しる.だから、動けるために、追加します。  #}
+{% block sub_title %}{% endblock %}
```

Comment:
This problem seems appear at `admin_change_password` only.
So you add this line at `change_password.twig` instead of here.